content/blog/2017/2017-02-10-blueocean-devlog-feb2.adoc

```diff
@@ -0,0 +1,71 @@
+---
+layout: post
+title: "Blue Ocean Dev Log: February Week #2"
+tags:
+- blueocean
+author: michaelneale
+---
+
+We're counting down the weeks until
+link:/projects/blueocea[Blue Ocean]
+1.0, which is planned for the end of March. If you hadn't picked up on the hint
+in my
+link:/blog/2017/02/03/blueocean-devlog-feb/[previous post],
+most of the Blue Ocean development team is in Australia, where it is currently
+the middle of summer. As I write this it is about 1000 degrees outside.
+Emergency measures such as air-conditioning and beer have been deployed in
+order to continue Blue Ocean development. This
+week featured a new beta with the
+link:/blog/2017/02/06/scm-api-2-take2/[SCM API]
+changes; many bug fixes, and some version bumps went out in beta 22. We also
+got some fresh new designs coming soon, though not in time for beta 22.
+
+image:/images/post-images/blueocean-dev-log/new-design-overview.png["Overview", role=center]
+
+Some development highlights:
+
+- Beta 22 went out featuring the new
+  link:https://github.com/jenkinsci/blueocean-plugin/commit/b8c8fb1080117f1c0448b0cfdb831543a109f4f8[SCM API]
+  with better use of GitHub API limits.
+- A fix for publishing of
+  link:https://wiki.jenkins-ci.org/display/JENKINS/PubSub+Light+Plugin[Server Side Events]
+  that made one CPU spin up to 100% was fixed (not good unless you want to heat up
+  your room)
+- Some new refinements to the design merged to the master branch (see images below).
+- Beta 22 featured the 1.0 version of
+  link:https://jenkins.io/blog/2017/02/03/declarative-pipeline-ga/[Declarative Pipeline]
+- An
+  link:https://github.com/jenkinsci/blueocean-plugin/commit/99524c36afedfb11150ac20d26c6b1d4e01b714a[Australian translation]
+  was added; really critical stuff, I know..
+- The Acceptance Test Harness (ATH) was stabilised a bit and it now covers
+  creating Pipelines from Git, which we talked about in
+  link:/blog/2017/01/27/blueocean-dev-log-jan4/[late January].
+- The plugin:blueocean-pipeline-editor[Visual Pipeline Editor] was released to the main Update Center
+  as a preview release, ready to play with!
```

Comment:
Maybe add a link or sub-bullet pointing to the public instance at https://ci.blueocean.io/blue/pipelines/pipeline-editor-preview/ ?